contracts/integrations/test/exchange/core_test.ts

```diff
@@ -249,115 +232,16 @@ blockchainTests.resets('Exchange core', () => {
             signedOrder = await orderFactory.newSignedOrderAsync();
         });
 
-        describe('callback signature types', () => {
```

Comment:
Why were these removed?

Reply:
they were moved to `signature_validator.ts` in `@0x/contracts-exchange` because they rely on `TestValidatorWalletContract`